src/Microsoft.VisualStudio.ProjectSystem.Managed.TestServices/Moq/AbstractMock.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Moq
+{
+    internal abstract class AbstractMock<T> : Mock<T>
+        where T : class
+    {
+        protected AbstractMock(MockBehavior behavior = MockBehavior.Loose)
```

Comment:
:bulb: Suggest defaulting to `MockBehavior.Strict`. It differs from the Moq default but provides a big advantage in test confidence. Moq defaults tend to violate post-conditions which is not a happy place.

Reply:
We started with strict originally in the project, and while it resulted a little more confidence - it's turned into convenience vs correctness. I found that it actively caused me to write less tests - because it was harder to get started.